docs/getting-started/how-to/use-github-codespaces-in-your-project/index.md

```diff
@@ -0,0 +1,155 @@
+This page explains how to work with OpenSAFELY projects using GitHub Codespaces.
+
+## Managing GitHub Codespaces
+
+### How to create a codespace
+
+To create a Codespace,
+[create a code repository](../../how-to/create-a-code-repository-for-your-project/index.md) first.
+
+Once you have a research code repository to work in:
+
+1. In a web browser,
+   go to your newly created research code repository on GitHub.
+1. Click the "Code" button
+1. Select "Create codespace on main".
+1. You will see a "Setting up your codespace" screen before the codespace is ready for use.
+   It may take a few minutes before the codespace loads
+   and displays the Visual Studio Code editor.
+
+!!! info
+    For "Create codespace on main",
+    "main" refers to the Git branch to use in the codespace.
+
+    If you are working on another repository branch,
+    it is possible to select "New from options",
+    which allows you to choose the branch.
```

Comment:
thought: I'm not sure if we want to mention branches at all in this page. There's a mention here, and further down when talking about pushing changes to the GitHub remote.